hex/spec/dependabot/hex/update_checker_spec.rb

```diff
@@ -359,7 +359,7 @@
       end
     end
 
-    context "with a dependency from a private repo" do
+    pending "with a dependency from a private repo" do
```

Comment:
Can you please add an explanatory code comment when making stuff like this pending?

I recently went through some pain on old tests that were marked pending w/o any comments trying to figure out why they were pending... and the test had been that way for several years, so the files had been restructured multiple times so it was extremely painful trying to navigate `git blame` to figure out why they were disabled... particularly when part of the file had been moved and part hadn't... and I wanted to ensure I understood why it was pending to ensure i didn't misunderstand what it was trying to test. So leaving the code comment in-line ensures it's clear to future devs why it's temp marked pending as that comment will follow the test even if it gets moved to a different file.